pkgs/top-level/qt5-packages.nix

```diff
@@ -164,8 +164,6 @@ in (noExtraAttrs (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdP
 
   libqaccessibilityclient = callPackage ../development/libraries/libqaccessibilityclient { };
 
-  kpmcore = callPackage ../development/libraries/kpmcore { };
```

Comment:
And this should probably stay as an alias because other things might be using it.

Reply:
There's already `// kdeGear` on top, so I guess `libsForQt5.kpmcore` works fine?

Reply:
Oh, yes.